Library/Homebrew/cmd/audit.rb

```diff
@@ -1235,3 +1232,31 @@ def problem(text)
     @problems << text
   end
 end
+
+# Debugging support
```

Comment:
I'm not really happy with this code being in the global scope. Can we make that nicer, maybe by drawing some inspiration [from `Homebrew::Diagnostic.inject_dump_stats!`](https://github.com/Homebrew/brew/blob/b53f0c5ada753d5ded6f6fbe4f920a4a43696394/Library/Homebrew/diagnostic.rb#L88-L101)?